eventsourcing/src/main/java/org/axonframework/eventsourcing/eventstore/inmemory/InMemoryEventStorageEngine.java

```diff
@@ -50,8 +50,19 @@
  */
 public class InMemoryEventStorageEngine implements EventStorageEngine {
 
-    private final NavigableMap<TrackingToken, TrackedEventMessage<?>> events = new ConcurrentSkipListMap<>();
-    private final Map<String, List<DomainEventMessage<?>>> snapshots = new ConcurrentHashMap<>();
+  protected final NavigableMap<TrackingToken, TrackedEventMessage<?>> events;
```

Comment:
We are using `4 spaces indentation`. It would be great if you could apply it to this change.